scripts/misc/gen-esbuild-test.mjs

```diff
@@ -15,7 +15,7 @@ import * as fsExtra from 'fs-extra'
 
 const TREE_SITTER_WASM_GO_FILENAME = path.resolve(
   import.meta.dirname,
-  '../../tmp/tree-sitter-go.wasm',
+  '../../temp/tree-sitter-go.wasm',
```

Comment:
It's `tmp`. `temp` is gonna be removed in `.gitignore`

Reply:
wait, if I change it to `tmp` then it would be stored in the git repository, does that violate what you said to download on demand?

Reply:
https://github.com/rolldown/rolldown/blob/d73d334564a19bf72ff0ece84d44766417f0ada7/.gitignore#L3

Reply:
ok